src/seq.rs

```diff
@@ -0,0 +1,119 @@
+use ChunkerImpl;
+
+/// A chunker that implements sequential content‐defined chunking using a
+/// configurable threshold of consecutive increasing or decreasing bytes.
+/// For this implementation, we focus on the _increasing_ case.
+/// If the desired sequence is not met after observing a given number of
+/// "opposing" slopes, the algorithm jumps forward a fixed number of bytes.
+///
+/// Source: Sreeharsha Udayashankar, Abdelrahman Baba, Samer Al-Kiswany: SeqCDC: Hashless Content-Defined 
+/// Chunking for Data Deduplication. MIDDLEWARE '24: Proceedings of the 25th International Middleware 
+/// Conference, Pages 292-298, https://doi.org/10.1145/3652892.3700766
+#[derive(Clone, Debug, Default)]
+struct SeqChunkerState {
+    /// The length of the current incrementing run.
+    increment_run_length: usize,
+
+    /// The previously ingested byte, if any.
+    previous_value: Option<u8>,
+
+    /// Current slope length.
+    slope_length: usize,
+
+    /// The position _relative to the last chunk boundary_.
+    /// Specifically, this resets to zero after a chunk is emitted.
+    /// We need this to track if we're done filling our window between calls to `find_boundary`,
+    /// since we're fed blocks of data which may be smaller than the window.
+    pos: usize,
```

Comment:
This probably needs an update to the doc comment, because it now tracks bytes left to skip.
Maybe also rename, not sure.